File: Python/Penjualan.py
from Komputer import Komputer

class Penjualan:
    def __init__(self, tanggal="", harga=0, komputer=None):
        self.tanggal = tanggal
        self.harga = harga
        self.komputer = komputer if komputer else Komputer()

    def set_tanggal(self, tanggal):
        self.tanggal = tanggal

    def set_harga(self, harga):
        self.harga = harga

    def set_komputer(self, komputer):
        self.komputer = komputer

    def get_tanggal(self):
        return self.tanggal

    def get_harga(self):
        return self.harga

    def get_komputer(self):
        return self.komputer

File: CPP/Penjualan.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komputer.cpp"

using namespace std;

class Penjualan {
private:
    string tanggal;
    int harga;
    Komputer komputer;
public:
    Penjualan()
    {


    }


    Penjualan(string tanggal, int harga, Komputer komputer)
    {
        this->tanggal = tanggal;
        this->harga = harga;
        this->komputer = komputer;
    }


    void settanggal(string tanggal)
    {
        this->tanggal = tanggal;
    }

    void setHarga(int harga)
    {
        this->harga = harga;
    }

    void setKomputer(Komputer Komponen)
    {
        this->komputer = komputer;
    }

    string getTanggal()
    {
        return tanggal;
    }

    int getHarga()
    {
        return harga;
    }

    Komputer getKomputer()
    {
        return this->komputer;
    }

    ~Penjualan()
    {




    }
};
